src/main/java/com/example/simplefileshare/simplefileshare/services/UploadInterface.java

```diff
@@ -0,0 +1,10 @@
+package com.example.simplefileshare.simplefileshare.services;
+import java.io.File;
+import org.springframework.web.multipart.MultipartFile;
+import com.amazonaws.SdkClientException;
+import java.io.FileNotFoundException;
+
+public interface UploadInterface {
+    public String uploadFile (File file) throws SdkClientException;
+    public String uploadMultipartFile (MultipartFile multipartFile) throws SdkClientException,FileNotFoundException;
```

Comment:
Please remove the exceptions from the interface as other implementations may not throw the same exceptions.

Reply:
okay.